<ui version="4.0" >
 <class>clockConfig</class>
 <widget class="QWidget" name="clockConfig" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="minimumSize" >
      <size>
       <width>548</width>
       <height>0</height>
      </size>
     </property>
     <property name="title" >
      <string/>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <widget class="QCheckBox" name="showSecondHandCheckBox" >
        <property name="toolTip" >
         <string>Show the second leds</string>
        </property>
        <property name="whatsThis" >
         <string>Check this if you want to display second leds in order to see the seconds.</string>
        </property>
        <property name="text" >
         <string>Show &amp;seconds</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="showGridCheckBox" >
        <property name="toolTip" >
         <string>Show the grid</string>
        </property>
        <property name="whatsThis" >
         <string>Check this if you want to see a grid around leds.</string>
        </property>
        <property name="text" >
         <string>Show grid</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="showOffLedsCheckBox" >
        <property name="toolTip" >
         <string>Show the off leds</string>
        </property>
        <property name="whatsThis" >
         <string>Check this if you want to see off leds.</string>
        </property>
        <property name="text" >
         <string>Show off leds</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer" >
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>20</width>
          <height>190</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <zorder>groupBox</zorder>
  <zorder>groupBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
